trafficFlowTests.py

```diff
@@ -129,6 +141,7 @@ def _run_test_case_instance(
         for t in servers + clients + monitors:
             t.initialize()
 
+        time.sleep(10)
```

Comment:
Why is there a 10 second sleep here?

Reply:
I also think that the sleep should be avoided.

It's a bit ugly that the `initialize()` part does not run in parallel (which maybe should be fixed). So creating the test pods takes already longer than necessary. But adding another sleep on top of that, increases the time waiting for the result.

Reply:
Added this initially, since EXTERNAL_IP tests were failing because of delay in docker container bring up. But the tests are now working with the latest changes and the latest image. So, removed this timeout now